seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -156,6 +156,9 @@ bnb,mdex,1,2022-12-01,0x5558ee58a68414ca812e6861e4f1743fd4670669ab67bdd6cfde72f2
 bnb,mdex,1,2022-12-01,0xfdd41f1fbd608530bd96c19fdfc713d861ec483d9c71976df981a4f61fd28140,376,0x55d398326f99059ff775485246999027b3197955,3.854107292216319,0x3ee2200efb3400fabb9aacf31297cbdd1d435d47,12.23
 bnb,mdex,1,2022-12-01,0x7d8f360ee960583133bbde85314d0257703feabf7a87518e5dc7b3a83de5c001,436,0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c,0.1,0x55d398326f99059ff775485246999027b3197955,29.673387128124077
 bnb,mdex,1,2022-12-01,0xc2e17fc3812f583f5cd2ac1622939828181600b4a65bc1a30449050e3955522c,341,0xe9e7cea3dedca5984780bafc599bd69add087d56,39.60004338983994,0xf8a0bf9cf54bb92f17374d9e9a321e6a111a51bd,5.179156079954205x
+bnb,ellipsis_finance,1,2022-05-22,0x85e68cfc48c650f4d7cfb9ce98b09c89e0952b3f7b7b2111ead705a72bd4cb38,349,0xe9e7cea3dedca5984780bafc599bd69add087d56,299.616793572788036088,0xdcecf0664c33321ceca2effce701e710a2d28a3f,314.194676267749213306
```

Comment:
The unmatched seed in dex_trades_seed is linked to the comment https://github.com/duneanalytics/spellbook/pull/2260#issuecomment-1357982123
Maybe we can check the progress of decoding and rerun the test job again.

Reply:
https://dune.com/queries/1836659?d=1